### DISCUSSION TITLE
Has anyone created a 'pipe' or 'cylinder' layer that follows a geojson polyline?

### DISCUSSION THREAD

**Discussion by bobfortin64** [visgl/deck.gl#8203] [General]
Has anyone created a 'pipe' or 'cylinder' layer that follows a geojson polyline?
I would like to visualize polylines -- eg: an underground sewere pipe or mining drillhole.

Has onyone done something similar? Likely properties would be:
extrude: bool
radius: double
radiusunits: string
resolution:  number // number of polygons to make up cylinder similar to columnlayer

-Bob

**Comment by linze99** [comment]
you can use a CompositeLayer which renders two SimpleMeshLayers.
One of the SimpleMeshLayers renders a CylinderGeometry for the lines of the pipe.
another one renders a SphereGeometry for the connection part of the pipe.
<img width="484" alt="2023-10-17 11 26 30" src="https://github.com/visgl/deck.gl/assets/30024479/abdfe936-7492-49e5-af99-a71a00dd8b54">
